acrobat/scripts/scripts.js

```diff
@@ -201,12 +99,13 @@ const CONFIG = {
   loadStyles(paths);
 
   // Import base milo features and run them
+  const { locales } = await import(`${miloLibs}/scripts/scripts.js`);
```

Comment:
this will make our page load slower.

Reply:
Change is in the microseconds. Will be loaded from `/scripts/locales.js` when change is made in Milo. This is POC right now.

Reply:
everything is microseconds.  hundreds of microseconds on a constrained connection.  If this is merged, won't it go live?